packages/react-ui-core/src/DateInput/DateInput.js

```diff
@@ -1,24 +1,33 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import DatePicker from 'react-datepicker'
+import DatePicker from 'react-basic-datepicker'
 
 export default class DateInput extends Component {
+  static propTypes = {
+    selected: PropTypes.object,
+    onChange: PropTypes.func,
+    maxDate: PropTypes.object,
+    minDate: PropTypes.object,
+    placeholderText: PropTypes.string,
+    className: PropTypes.string,
+    name: PropTypes.string,
+  }
 
- static propTypes = {
-   selected: PropTypes.object,
-   onChange: PropTypes.func,
-   maxDate: PropTypes.object,
-   minDate: PropTypes.object,
-   placeholderText: PropTypes.string,
- }
+  static defaultProps = {
+    placeholderText: 'Date',
+  }
 
- static defaultProps = {
-   placeholderText: 'Date',
- }
+  render() {
+    const { selected, onChange, className, name, ...rest } = this.props
 
- render() {
-   return (
-     <DatePicker {...this.props} />
-   )
- }
+    return (
+      <DatePicker
+        handleDateChange={onChange}
+        value={selected}
+        datepickerName={name}
+        datepickerClassName="DatePicker"
```

Comment:
This component is `DateInput` - should the classname also be `DateInput` for consistency?

Reply:
Actually, this is the classname for the picker itself - that goes to the child, since it's datepickerClassName, rather than className. I think it makes sense like this, but we could always change it

Reply:
Is there a way to pass in whatever you want? Like could we theme this component?